asset-transfer-basic/application-javascript/app.js

```diff
@@ -128,8 +128,13 @@ async function main() {
 			// This will be sent to both peers and if both peers endorse the transaction, the endorsed proposal will be sent
 			// to the orderer to be committed by each of the peer's to the channel ledger.
 			console.log('\n--> Submit Transaction: CreateAsset, creates new asset with ID, color, owner, size, and appraisedValue arguments');
-			await contract.submitTransaction('CreateAsset', 'asset13', 'yellow', '5', 'Tom', '1300');
-			console.log('*** Result: committed');
+			result = await contract.submitTransaction('CreateAsset', 'asset13', 'yellow', '5', 'Tom', '1300');
+			// Notice how we have a result on the submit, this will be returned by the chaincode when
+			// preforming the function 'CreateAsset'. Only the javascript version of the assetTransfer basic chaincode
```

Comment:
Can you explain "Only the javascript version..."

Is there some reason it can't be done in the other versions? Or we just choose not to do it?

Also, typo in "preforming".